code/modules/religion/sects/shadow_sect.dm

```diff
@@ -42,8 +43,13 @@
 
 /datum/religion_sect/shadow_sect/on_select(atom/religious_tool, mob/living/user)
 	. = ..()
+	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
 	if(!religious_tool || !user)
 		return
+	if(user.mind.is_murderbone())
+		sect.Murder = TRUE
+	else
+		sect.Murder = FALSE
```

Comment:
By the way, should ``Murder`` be capitalized?
```suggestion
	sect.Murder = user.mind.is_murderbone()
```

Reply:
Capitalised as its cap'd in the var, but I can change that if needed

Reply:
Sorry, I meant all instances
Typically, per our style, ``variables_arent_capitalized``